source/backend/api/Helpers/Healthchecks/PimsExternalApiHealthcheck.cs

```diff
@@ -0,0 +1,66 @@
+using System;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Pims.Core.Exceptions;
+
+namespace Pims.Api.Helpers.Healthchecks
+{
+    public class PimsExternalApiHealthcheck : IHealthCheck
```

Comment:
The idea here is that this is a generic health check that we can use for any external, unauthenticated api service.